src/dotnet/commands/dotnet-tool/uninstall/LocalizableStrings.resx

```diff
@@ -144,19 +144,31 @@
   <data name="FailedToUninstallTool" xml:space="preserve">
     <value>Failed to uninstall tool '{0}': {1}</value>
   </data>
-  <data name="UninstallToolCommandNeedGlobalOrToolPath" xml:space="preserve">
-    <value>Please specify either the global option (--global) or the tool path option (--tool-path).</value>
-  </data>
   <data name="ToolPathOptionName" xml:space="preserve">
     <value>PATH</value>
   </data>
   <data name="ToolPathOptionDescription" xml:space="preserve">
     <value>The directory containing the tool to uninstall.</value>
   </data>
-  <data name="UninstallToolCommandInvalidGlobalAndToolPath" xml:space="preserve">
-    <value>(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</value>
-  </data>
   <data name="InvalidToolPathOption" xml:space="preserve">
     <value>Tool path '{0}' does not exist.</value>
   </data>
-</root>
\ No newline at end of file
+    <data name="ManifestPathOptionDescription" xml:space="preserve">
+    <value>Path to the manifest file.</value>
+  </data>
+  <data name="ManifestPathOptionName" xml:space="preserve">
+    <value>PATH</value>
+  </data>
+  <data name="LocalOptionDescription" xml:space="preserve">
+    <value>Uninstall the tool and remove it from the local tool manifest.</value>
+  </data>
+  <data name="UninstallToolCommandInvalidGlobalAndLocalAndToolPath" xml:space="preserve">
+    <value>The following options conflict: {0}. Use only one of the following options: the local option (--local), the global option (--global), or the tool path option (--tool-path).</value>
+  </data>
+  <data name="OnlyLocalOptionSupportManifestFileOption" xml:space="preserve">
```

Comment:
Specifying the tool manifest (--tool-manifest) is only valid with the local option (--local or the default).